ci-operator/jobs/stackrox/stackrox/stackrox-stackrox-release-3.74-presubmits.yaml

```diff
@@ -0,0 +1,1688 @@
+presubmits:
+  stackrox/stackrox:
+  - agent: kubernetes
+    always_run: true
```

Comment:
```suggestion
    always_run: false
```

Reply:
Thanks! I forgot about this. To not run into such thing next time, I put info to Confluence 

https://docs.engineering.redhat.com/display/StackRox/Making+changes+to+OpenShift+CI#MakingchangestoOpenShiftCI-PullRequestchecklist

Is there anything else you usually check in generated files that's worth mentioning?

Reply:
Done.